ansible_collections/arista/avd/roles/eos_designs/doc/l3-edge.md

```diff
@@ -11,6 +11,7 @@ Make sure to configure the variables in a group_vars file covering all devices m
 l3_edge:
   p2p_links_ip_pools:
     < p2p_pool_name_1 >: < IPv4_address/Mask >
+      ipv4_pool_mask: < subnet mask size | default -> 31 >
```

Comment:
this looks weird to have both `< IPv4_address/Mask >` and `ipv4_pool_mask` close to each other. Maybe let's use prefix_size ?

Reply:
will change to prefix_size